src/missions/use-after-free-control-flow/temporal-mission.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ * Copyright (c) 2020 Microsoft, Inc.
+ */
+#include <assert.h>
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <strings.h>
+#include <unistd.h>
+
+#include <sys/queue.h>
+
+#if defined(CAPREVOKE) && __CHERI_PURE_CAPABILITY__
+#include <sys/caprevoke.h>
+static void __attribute__((used)) *check_caprevoke = caprevoke;
+extern void malloc_revoke(void);
+static void __attribute__((used)) *check_malloc_revoke = malloc_revoke;
+#endif
+
+struct farm;
+struct crop;
+
+static void
+success(struct farm *f)
+{
+	fprintf(stderr, "Exploit successful: system bought the farm!\n");
+	exit(0);
```

Comment:
If only this were GNU/Hurd, then we could exit with EIEIO.

Reply:
For maximal confusion, one day I hope to see HURD running on POWER.